tests/ha/qnetd.pm

```diff
@@ -29,19 +29,20 @@ sub qdevice_status {
     $num_nodes-- if ($expected_status eq 'stopped');
 
     # Check qdevice status
-    script_run "$qnetd_status_cmd";
+    $output = script_output "$qnetd_status_cmd", proceed_on_failure => 1;
```

Comment:
We could also remove the `, proceed_on_failure => 1` and instead use `if ($expected_status ne 'stopped')`. @ldevulder @juadk what do you think would be better?

Reply:
Maybe `if ($expected_status ne 'stopped')` is more readable.

Reply:
If it works yes, it's more readable.

Reply:
OK. Will push this change and add a new verification run.